pwiz_tools/Skyline/SkylineFiles.cs

```diff
@@ -1871,7 +1871,7 @@ public void ImportMassList(string fileName)
             }
         }
 
-        public void ImportMassList(MassListInputs inputs, string description, bool assayLibrary)
+        public void ImportMassList(MassListInputs inputs, string description, bool assayLibrary, SrmDocument.DOCUMENT_TYPE inputType = SrmDocument.DOCUMENT_TYPE.none)

```

Comment:
General need for comment headers throughout, for the next coder that comes along